pwndbg/commands/heap.py

```diff
@@ -80,41 +87,59 @@ def bins(addr=None):
     fastbins = main_arena['fastbinsY']
     bins = main_arena['bins']
 
+    size_t_size = gdb.lookup_type('size_t').sizeof
```

Comment:
And in before "they're the same", see x32 ABI with 4-byte pointers and 8-byte size_t 😛